training/controllers/basic_funcionalities.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, session, g
+
+from training.controllers.function_decorators import login_required
+
+#Hay que ver donde carajo poner esto--------------------------------------------------------------------------------------
+# from training.controllers.admin import admin_web
+#Hay que ver donde carajo poner esto--------------------------------------------------------------------------------------
+
+
+bp = Blueprint('name2', __name__)
+
+
+@bp.route('/inside')
+@login_required
+def inside():
+    if 'username' in session:
+        return "You are already logged as --> " + str(session['username']), 200
+
+    return "You are not logged in", 400
```

Comment:
if login es required for this view I think it'll never reach this point and you can assume user will be present

Reply:
That is true, changed!